src/recipes/lust-for-life.json

```
@@ -0,0 +1,51 @@
+{
+    "name": "Lust For Life",
+    "description": "This exotic sipper from Imbibe’s 2013 Cocktail Bar of the Year, Pouring Ribbons, is a little smoky, a little nutty and perfectly bright",
+    "github": "Donearm",
+
+    "ingredients": [
+        {
+            "quantity": "45",
+            "measure": "ml",
+            "ingredient": "galliano"
+        },
+        {
+            "quantity": "30",
+            "measure": "ml",
+            "ingredient": "orange juice"
+        },
+        {
+            "quantity": "15",
+            "measure": "ml",
+            "ingredient": "peach liqueur"
+        },
+        {
+            "quantity": "15",
+            "measure": "ml",
+            "ingredient": "heavy cream"
+        },
+		{
+			"quantity": "",
+			"measure": "",
+			"ingredient": "ice cubes"
+		},
+        {
+            "quantity": "",
+            "measure": "",
+            "ingredient": "grated nutmeg"
+        }
+    ],
+    "directions": [
+		"In a shaker, combine ingredients and shake with ice until chilled.",
+		"Strain into a rocks glass on ice cubes.",
+		"Garnish with grated nutmeg."
+    ],
+    "image": "lustforlife.jpg",
```

```suggestion
    "image": "lust-for-life.jpg",
```
Can the image filename match the recipe filename?

Also, if you can provide a horizontal image at 600x400px or square.

---

Changed filename. As for the image, I couldn't find a better one but I resized the one to be nearly perfectly square.